mods/default/craftitems.lua

```diff
@@ -20,20 +20,20 @@ local function book_on_use(itemstack, user, pointed_thing)
 	end
 	local formspec
 	if owner == player_name then
-		formspec = "size[8,8]"..default.gui_bg..
-			default.gui_bg_img..
-			"field[0.5,1;7.5,0;title;Title:;"..
-				minetest.formspec_escape(title).."]"..
-			"textarea[0.5,1.5;7.5,7;text;Contents:;"..
-				minetest.formspec_escape(text).."]"..
+		formspec = "size[8,8]" .. default.gui_bg ..
+			default.gui_bg_img ..
+			"field[0.5,1;7.5,0;title;Title:;" ..
+				minetest.formspec_escape(title) .. "]" ..
+			"textarea[0.5,1.5;7.5,7;text;Contents:;" ..
+				minetest.formspec_escape(text) .. "]" ..
 			"button_exit[2.5,7.5;3,1;save;Save]"
 	else
-		formspec = "size[8,8]"..default.gui_bg..
-			default.gui_bg_img..
-			"label[0.5,0.5;by "..owner.."]"..
-			"label[0.5,0;"..minetest.formspec_escape(title).."]"..
-			"textarea[0.5,1.5;7.5,7;text;;"..
- 				minetest.formspec_escape(text).."]"
+		formspec = "size[8,8]" .. default.gui_bg ..
+			default.gui_bg_img ..
+			"label[0.5,0.5;by " .. owner .. "]" ..
```

Comment:
It isn't necessary, since player names are very restricted, but I'd add it anyways in case the list of player name characters is ever extended.